bundles/cis-k8s-1.5.1/5.1.1_restrict-clusteradmin-rolebindings.yaml

```diff
@@ -18,8 +18,8 @@ metadata:
   name: restrict-clusteradmin-rolebindings
   annotations:
     # This constraint is not certified by CIS.
-    description: "Restricts use of the cluster-admin role."
-    bundles.validator.forsetisecurity.org/cis-k8s-v1.5.1: 5.1.1
+    description: Restricts use of the cluster-admin role.
+    bundles.validator.forsetisecurity.org/cis-k8s-1.5.1: 5.1.1
```

Comment:
Why removing the v? It's helpful to separate the version from the name.

Reply:
I agree but the folder doesn't have the "v" in there which complicates the copying process from the internal library. We can either take the v out from the annotation or put a "v" in the directory. I thought the annotation modification may be less obtrusive

Reply:
Added